src/identity/allowances/actions/thunks/index.ts

```diff
@@ -0,0 +1,38 @@
+// Libraries
+import {Dispatch} from 'react'
+
+// Actions
+import {
+  Actions as OrgCreationAllowancesActions,
+  setOrgCreationAllowances,
+  setOrgCreationAllowancesStatus,
+} from 'src/identity/allowances/actions/creators'
+
+// API
+import {fetchOrgCreationAllowance} from 'src/identity/apis/org'
+
+// Types
+import {GetState, RemoteDataState} from 'src/types'
+
+// Utils
+import {reportErrorThroughHoneyBadger} from 'src/shared/utils/errors'
+
+type Actions = OrgCreationAllowancesActions
+
+export const getOrgCreationAllowancesThunk =
+  () => async (dispatch: Dispatch<Actions>, getState: GetState) => {
+    try {
+      dispatch(setOrgCreationAllowancesStatus(RemoteDataState.Loading))
+
+      const allowances = await fetchOrgCreationAllowance()
+
+      dispatch(setOrgCreationAllowances(allowances))
+
+      dispatch(setOrgCreationAllowancesStatus(RemoteDataState.Done))
+    } catch (error) {
+      reportErrorThroughHoneyBadger(error, {
```

Comment:
Once we hear back from AIM, let's see if we can report only errors that aren't a `401` status code (we can skip error messages that are an artifact of the user being logged out).

Reply:
yes, I didn't forget about our discussion on filtering out 401s from being sent to honeybadger. are they working on adding 401 status to the api response?